database/seeds/UserTableSeeder.php

```diff
@@ -50,7 +50,7 @@ public function run()
         // Add Contestant Users
         $waitingRooms = WaitingRoom::all();
         $totalRooms = count($waitingRooms);
-        $seedContestants = $this->northstar->getSeedUsers(300);
+        $seedContestants = $this->northstar->getUsers(100);
```

Comment:
not a blocker for this PR, but did we wanna add my loop thing to the seed file, or is 100 users sufficient?